runtime/syntax/asciidoc.vim

```diff
@@ -25,6 +25,25 @@ if exists("b:current_syntax")
   finish
 endif
 
+" Conceal Systems
+let s:conceal = ''
+let s:concealends = ''
+let s:concealcode = ''
+if has('conceal') && get(g:, 'vim_asciidoc_conceal', 1)
```

Comment:
missing documented variable

Reply:
This is parameter to use build in vim syn-concealends and syn-conceal to active on specific region.
first vim check build in conceal feature. then if let g:vim_asciidoc_conceal=0 vim will not activated conceal in specific region.